README.md

```diff
@@ -94,6 +94,7 @@ Command                                | List                               | Re
 `Clap lines`                           | Lines in the loaded buffers        | _none_
 `Clap marks`                           | Marks                              | _none_
 `Clap tags`                            | Tags in the current buffer         | **[vista.vim][vista.vim]**
+`Clap yanks`                           | Yank stack of the current vim session    | ViM(8.0.1394+)/NeoVim(0.3.0+)
```

Comment:
I believe the requirement has been covered by clap itself.

Reply:
Yes, adjusted accordingly.